.github/workflows/docker-build.yml

```diff
@@ -6,9 +6,20 @@ on:
     tags:
       - "*"
     paths-ignore:
+      - ".circleci"
+      - ".github"
+      - "!.github/workflows/docker-build.yml"
       - ".gitignore"
       - "**.md"
-      - "test/"
+      - "**test**"
+      - "docs/"
+      - "**vcpkg**"
+      - "run_route_scripts/"
+      - ".*"
+      - "pyproject.toml"
+      - "*.cfg"
+      - "setup.py"
+      - "taginfo.json"
```

Comment:
```suggestion
      - "**test**"
      - "docs/"
      - "**vcpkg**"
      - "run_route_scripts/"
      - "test_requests/"
      - ".*"
      - "pyproject.toml"
      - "*.cfg"
      - "setup.py"
      - "taginfo.json"
```

Reply:
`"**test**"` does that already. though that might actually be a bit very greedy..